ui/radar/radar.js

```diff
@@ -292,8 +296,7 @@ class Radar {
 
     // change instance
     if (type === '00') {
-      const lang = this.options.ParserLanguage;
-      if (log.match(instanceChangedRegex[lang] || instanceChangedRegex['en'])) {
+      if (this.instanceChangedRegex.test(log)) {
```

Comment:
I'm assuming this was a bug since the latter part of the `or` was testing for the English regex existence, so I changed that but if it was intentional I'll revert that :upside_down_face: